google/datalab/bigquery/commands/_bigquery.py

```diff
@@ -376,17 +376,45 @@ def _get_query_parameters(args, cell_body):
   if config:
     jsonschema.validate(config, BigQuerySchema.QUERY_PARAMS_SCHEMA)
 
+    # We merge the parameters with the airflow macros so that users can specify certain airflow
+    # macro names (like '@ds') in their sql. This is useful for enabling the user to progressively
+    # author a pipeline with %bq pipeline.
+
+    import datetime
+    canned_parameters = {
```

Comment:
As I look at this, I wonder if these should be prefixed in some way to avoid potential conflicts with user-defined parameters having similar name. Example... `ds` and `ts` don't seem like the ones potentially already in use.